bin/requirements.txt

```diff
@@ -1,7 +1,7 @@
 bitstring==3.1.9
-constraint==0.4.1
+#constraint==0.4.1
 dv==1.0.10
-Jinja2==2.11.3
+Jinja2==3
```

Comment:
I have momentarily seen an error about jinja2 myself recently, but then I had not configured my shell properly.
Since our IT setup is different, I don't anticipate that this change will affect us.

Is the usage of jinja2 within the python scripts still functioning after upgrading to major version 3 like this?
If yes, then I do not oppose the change.

Reply:
I'm checking the status of Jinja changes on the CVA6 side, will report the feedback here when available.

Reply:
> I'm checking the status of Jinja changes on the CVA6 side, will report the feedback here when available.

No adverse impact in CVA6 (already using `Jinja2 v3.x`, not using `constraint`) ==> LGTM.